### PR TITLE
Adds a `repository` and `homepage` field to `package.json`

### PR DESCRIPTION
**Pull request by foot**
- So we get a link back to this repo from the npm page

### CODE REVIEW

File: plugins/backstage-plugin-flux/package.json [hunk 1]
```diff
@@ -4,6 +4,12 @@
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "MPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/weaveworks/weaveworks-backstage.git",
+    "directory": "plugins/backstage-plugin-flux"
+  },
+  "homepage": "https://docs.gitops.weave.works/docs/backstage/intro/",
```
Comment (foot) [inline plugins/backstage-plugin-flux/package.json:12]: Not sure what this should be, we could also omit if this doesn't feel quite right.

Comment (bigkevmcd) [reply plugins/backstage-plugin-flux/package.json:12]: Given how often we move our docs around, maybe we leave it out?

Comment (bigkevmcd) [reply plugins/backstage-plugin-flux/package.json:12]: Or link to the GitHub Repo?

**Review by bigkevmcd — APPROVED** [review]
Let's fix that homepage key in some way, otherwise 👍🏻

**Pull request merged**